src/java.base/share/classes/java/io/Console.java

```diff
@@ -93,7 +96,7 @@
  * @since   1.6
  */
 
-public final class Console implements Flushable
+public class Console implements Flushable
```

Comment:
Should we perhaps `seal` this class and only `permit` `ProxyingConsole` to `extend` it?

Reply:
Right. Will address it after this PR.

Reply:
Filed: https://bugs.openjdk.org/browse/JDK-8298416